src/contributing/style_guide.md

```diff
@@ -0,0 +1,85 @@
+# Style Guide
+
+These are some general style guidelines for articles which appear on `docs.telemetry.mozilla.org` (DTMO). Reading this will help you write material that is more useful to others.
+
+<!-- toc -->
+
+## Audience
+
+The primary audience for DTMO are *data practitioners* at Mozilla. A "data practitioner" is someone who wants to inform themselves or others using Mozilla-collected data. Here are some real-world examples of this persona:
+
+* A data scientist performing an experiment analysis
+* An analyst producing a report on the effectiveness of a recent marketing campaign
+* A Firefox engineer trying to measure the performance of a new feature
+* A technical product manager trying to understand the characteristics of a particular user segment
+* A quality assurance engineer trying to understand the severity and frequency of a new Firefox crash
+
+In general, you can assume that readers have at least some technical knowledge. Different articles on DTMO may have different target audiences: when writing a new article, you should consider who you are writing it for and adjust your content appropriately. For example, a new product manager may require more careful hand-holding (and links to relevant concept or reference material) than a data scientist with many years of experience at Mozilla.
```

Comment:
audiences. When you write a new article, you should consider for whom you are writing it and adjust the content appropriately.